auth0_flutter/test/web/auth0_flutter_web_test.mocks.dart

```diff
@@ -46,6 +48,18 @@ class _FakeWebCredentials_1 extends _i1.SmartFake
         );
 }
 
+@JSExport()
+class _FakeRedirectLoginResult_2 extends _i1.SmartFake
```

Comment:
I had to add this manually, since mockito would crash. Is a known issue, though.
https://github.com/dart-lang/mockito/issues/762